core/src/test/java/com/linecorp/armeria/server/management/ManagementServiceTest.java

```diff
@@ -103,4 +108,22 @@ void heapDump() throws InterruptedException {
         // Make sure that the returned file has a valid hprof format
         assertThat(Arrays.copyOf(actual, fileHeader.length)).isEqualTo(fileHeader);
     }
+
+    @Test
+    void appInfo() throws JsonProcessingException {
+        final WebClient client = WebClient.builder(server.httpUri())
+                                          .build();
+
+        final AggregatedHttpResponse response = client.get("/internal/management/info").aggregate().join();
+        final String content = response.contentUtf8();
+        final JsonNode jsonNode = mapper.readTree(content).path("app");
```

Comment:
Optional) We could use `assertThatJson` for fluent validations for json objects

Reply:
Can you also validate that the `armeria` node is also serialized/deserialized correctly?